### DISCUSSION TITLE
关于去掉论文中的《学位论文独创性申明》和《授权声明》

### DISCUSSION THREAD

**Discussion by Excellence-creator** [sikouhjw/gdutthesis#69] [Q&A] [answered]
怎么暂时去掉《学位论文独创性申明》和《授权声明》，因为查重会查到这部分。
![Uploading image.png…]()

**Comment by sikouhjw** [comment, marked as answer]
注释
https://github.com/sikouhjw/gdutthesis/blob/c3cc2de5bfa47f7bf1b88c0884cca60119d6fd82/main.tex#L219
